modules/planning/tuning/autotuning_mlp_net_model.h

```diff
@@ -39,6 +39,23 @@ class AutotuningMLPModel : public prediction::network::NetModel {
    */
   void Run(const std::vector<Eigen::MatrixXf>& inputs,
            Eigen::MatrixXf* output) const override;
+
+  /**
+   * @brief Set the internal state of a network model
+   * @param A specified internal state in a vector of Eigen::MatrixXf
+   */
+   void SetState(const std::vector<Eigen::MatrixXf>& states)override{}
```

Comment:
It might be better to revert to declaring the State related  functions as virtual.
Alternatively we could implement these methods and use them in unit tests.